### PR TITLE
profile 1: initial profile to help unblock revoke

### PR DESCRIPTION
**Pull request by chrisnojima**
@awendland this should help unblock the revoke stuff
@keybase/react-hackers this is just the initial part of the integration . leading us to the username screen and thats it

### CODE REVIEW

File: shared/actions/profile.js [hunk 1]
```diff
@@ -42,3 +49,91 @@ export function editProfile (bio: string, fullname: string, location: string) :
...
+        service: platform,
+        username: '',
+        force: true,
+        promptPosted: false,
+        auto: false,
+      },
+      incomingCallMap: {
+        'keybase.1.proveUi.promptUsername': ({prompt, prevError}, response) => {
+          submitUsernameResponse = response
+          dispatch(routeAppend({path: 'ProveEnterUsername'}))
```
Comment (chrisnojima) [inline shared/actions/profile.js]: using the string based navigation stuff / entirely store based action except for the stashing of the response object

File: shared/profile/index.js [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 import {getProfile, updateTrackers} from '../actions/tracker'
-import {routeAppend, navigateUp} from '../actions/router'
-import {openInKBFS} from '../actions/kbfs'
 import {isLoading} from '../constants/tracker'
+import {openInKBFS} from '../actions/kbfs'
+import {routeAppend, navigateUp} from '../actions/router'
+import ProveEnterUsername from './prove-enter-username-container'
 
 class Profile extends Component<void, Props, void> {
   static parseRoute (currentPath, uri) {
```
Comment (chrisnojima) [inline shared/profile/index.js]: originally i didn't need this component but i do for the parseRoute function. when we revisit the meta-navigator we could remove this requirement then we could get rid of this wrapper

File: shared/profile/prove-enter-username-container.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+// @flow
+import React, {Component} from 'react'
+import ProveEnterUsername from './prove-enter-username'
+import {bindActionCreators} from 'redux'
+import {connect} from 'react-redux'
+import {submitUsername, cancelAddProof, updateUsername} from '../actions/profile'
+
+class ProveEnterUsernameContainer extends Component<void, any, void> {
```
Comment (chrisnojima) [inline shared/profile/prove-enter-username-container.js]: originally i didn't need this component but i do for the parseRoute function. when we revisit the meta-navigator we could remove this requirement then we could get rid of this wrapper

File: shared/profile/prove-enter-username-container.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+// @flow
```
Comment (chrisnojima) [inline shared/profile/prove-enter-username-container.js]: i named this -container as these classes are normally called -render but they weren't named that.

**Comment by MarcoPolo** [comment]
Should we review this or wait?

**Comment by chrisnojima** [comment]
wait

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 268b023a8de0...1450095fad8b introduced some visual changes on linux:
 * Added: **Revoke_Proof-Twitter_Waiting**  
   Revoke_Proof-Twitter_Waiting rendered
 * Added: **Revoke_Proof-Twitter_Error**  
   Revoke_Proof-Twitter_Error rendered
 * Added: **New_Proof_Enter_Username-Twitter_with_Error**  
   New_Proof_Enter_Username-Twitter_with_Error rendered
 * Added: **New_Proof_Enter_Username-Coinbase_with_Error**  
   New_Proof_Enter_Username-Coinbase_with_Error rendered
 * Changed: **Tracker-New_user_Scroll_1** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Tracker-New_user_Scroll_1-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Tracker-New_user_Scroll_1-1450095fad8b.png)  
   Tracker-New_user_Scroll_1 comparison
 * Changed: **Revoke_Proof-https_website** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-https_website-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-https_website-1450095fad8b.png)  
   Revoke_Proof-https_website comparison
 * Changed: **Revoke_Proof-Website** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Website-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Website-1450095fad8b.png)  
   Revoke_Proof-Website comparison
 * Changed: **Revoke_Proof-Twitter** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Twitter-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Twitter-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Twitter.png)
 * Changed: **Revoke_Proof-Reddit** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Reddit-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Reddit-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Reddit.png)
 * Changed: **Revoke_Proof-Hacker_News** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Hacker_News-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Hacker_News-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Hacker_News.png)
 * Changed: **Revoke_Proof-Git_Hub** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Git_Hub-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Git_Hub-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Git_Hub.png)
 * Changed: **Revoke_Proof-DNS** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-DNS-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-DNS-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-DNS.png)
 * Changed: **Revoke_Proof-Coinbase** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Coinbase-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Coinbase-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Coinbase.png)
 * Changed: **Revoke_Proof-Bitcoin** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Bitcoin-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Bitcoin-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Revoke_Proof-Bitcoin.png)
 * Changed: **Profile-Your_Profile_Proof_Menu** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Proof_Menu-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Proof_Menu-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Proof_Menu.png)
 * Changed: **Profile-Your_Profile_Pending_Proof_Menu_HN** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Pending_Proof_Menu_HN-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Pending_Proof_Menu_HN-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Pending_Proof_Menu_HN.png)
 * Changed: **Profile-Your_Profile_Pending_Proof_Menu_DNS** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Pending_Proof_Menu_DNS-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Pending_Proof_Menu_DNS-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Pending_Proof_Menu_DNS.png)
 * Changed: **Profile-Your_Profile_Pending_Proof_Menu** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Pending_Proof_Menu-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Pending_Proof_Menu-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Pending_Proof_Menu.png)
 * Changed: **Profile-Your_Profile_No_Proofs** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_No_Proofs-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_No_Proofs-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_No_Proofs.png)
 * Changed: **Profile-Your_Profile_Empty** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Empty-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Empty-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Empty.png)
 * Changed: **Profile-Your_Profile_Broken_Proof_Menu** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Broken_Proof_Menu-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Broken_Proof_Menu-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Broken_Proof_Menu.png)
 * Changed: **Profile-Your_Profile_Broken** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Broken-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Broken-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile_Broken.png)
 * Changed: **Profile-Your_Profile** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Profile-Your_Profile.png)
 * Changed: **New_Proof_Post-Twitter_Waiting** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Twitter_Waiting-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Twitter_Waiting-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Twitter_Waiting.png)
 * Changed: **New_Proof_Post-Twitter_Error** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Twitter_Error-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Twitter_Error-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Twitter_Error.png)
 * Changed: **New_Proof_Post-Twitter** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Twitter-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Twitter-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Twitter.png)
 * Changed: **New_Proof_Post-Reddit** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Reddit-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Reddit-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Reddit.png)
 * Changed: **New_Proof_Post-Hacker_News** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Hacker_News-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Hacker_News-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Hacker_News.png)
 * Changed: **New_Proof_Post-HTTP** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-HTTP-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-HTTP-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-HTTP.png)
 * Changed: **New_Proof_Post-Git_Hub** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Git_Hub-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Git_Hub-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Git_Hub.png)
 * Changed: **New_Proof_Post-DNS** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-DNS-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-DNS-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-DNS.png)
 * Changed: **New_Proof_Post-Coinbase** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Coinbase-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Coinbase-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Post-Coinbase.png)
 * Changed: **New_Proof_Enter_Username-Website** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Website-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Website-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Website.png)
 * Changed: **New_Proof_Enter_Username-Twitter** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Twitter-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Twitter-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Twitter.png)
 * Changed: **New_Proof_Enter_Username-Reddit** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Reddit-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Reddit-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Reddit.png)
 * Changed: **New_Proof_Enter_Username-Hacker_News** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Hacker_News-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Hacker_News-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Hacker_News.png)
 * Changed: **New_Proof_Enter_Username-Git_Hub** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Git_Hub-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Git_Hub-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Git_Hub.png)
 * Changed: **New_Proof_Enter_Username-DNS** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-DNS-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-DNS-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-DNS.png)
 * Changed: **New_Proof_Enter_Username-Coinbase** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Coinbase-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Coinbase-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Coinbase.png)
 * Changed: **New_Proof_Enter_Username-Bitcoin_Disabled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Bitcoin_Disabled-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Bitcoin_Disabled-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Bitcoin_Disabled.png)
 * Changed: **New_Proof_Enter_Username-Bitcoin** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Bitcoin-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Bitcoin-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Enter_Username-Bitcoin.png)
 * Changed: **New_Proof_Confirm_or_Pending-Pending_dns** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Pending_dns-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Pending_dns-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Pending_dns.png)
 * Changed: **New_Proof_Confirm_or_Pending-Pending_Hacker_News** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Pending_Hacker_News-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Pending_Hacker_News-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Pending_Hacker_News.png)
 * Changed: **New_Proof_Confirm_or_Pending-Confirm_http** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_http-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_http-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_http.png)
 * Changed: **New_Proof_Confirm_or_Pending-Confirm_Twitter** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Twitter-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Twitter-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Twitter.png)
 * Changed: **New_Proof_Confirm_or_Pending-Confirm_Reddit** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Reddit-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Reddit-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Reddit.png)
 * Changed: **New_Proof_Confirm_or_Pending-Confirm_Git_Hub** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Git_Hub-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Git_Hub-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Git_Hub.png)
 * Changed: **New_Proof_Confirm_or_Pending-Confirm_Coinbase** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Coinbase-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Coinbase-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Coinbase.png)
 * Changed: **New_Proof_Confirm_or_Pending-Confirm_Bitcoin** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Bitcoin-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Bitcoin-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/New_Proof_Confirm_or_Pending-Confirm_Bitcoin.png)
 * Changed: **Icon-Icon_Image** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Icon-Icon_Image-268b023a8de0.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Icon-Icon_Image-1450095fad8b.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1470766969584-268b023a8de0-1450095fad8b-linux/Icon-Icon_Image.png)

**Comment by chrisnojima** [comment]
replaced by https://github.com/keybase/client/pull/3802

**Pull request closed without merging**